src/Enum/DatasetLifecycleStatus.php

```diff
@@ -11,6 +11,8 @@ enum DatasetLifecycleStatus: string
     case RESTRICTED = 'Restricted';
     case SUBMITTED = 'Submitted';
     case IDENTIFIED = 'Identified';
+    case INCOMPLETE = 'Incomplete';
+    case PENDING = 'Pending';
```

Comment:
Maybe something that describes the state better, like Difpending would work. I'm a bit concerned it would be confusing to remember that this is a dif-only phase without it being in the name.